crates/storage/provider/src/providers/state/historical.rs

```diff
@@ -272,6 +275,24 @@ impl<'b, TX: DbTx> StateRootProvider for HistoricalStateProviderRef<'b, TX> {
             .state_root_with_updates(self.tx)
             .map_err(|err| ProviderError::Database(err.into()))
     }
+
+    fn storage_root_from_range(
+        &self,
+        address: Address,
+        range: RangeInclusive<BlockNumber>,
+    ) -> ProviderResult<B256> {
```

Comment:
@rkrasiuk do we need the range or a single block?

from rpc pov we need the root at a specific block

Reply:
you need the revert range, but the end block should always be latest, so we can just do a single block